src/components/icon/index.tsx

```diff
@@ -27,6 +27,15 @@ export type IconProps = Omit<ImageProps, 'source'> &
      * whether the icon should flip horizontally on RTL
      */
     supportRTL?: boolean;
+    /**
+     * 
+     * The icon width, width will be used only in case that the icon is svg image.
+     */
+    width?: string | number;
+    /**
+     * The icon height, height will be used only in case that the icon is svg image.
+     */
+    height?: string | number;
```

Comment:
Why did you decide to add the types (which will only work for SVG) and not set width\height from size to the SVG image?

Reply:
Since RN deprecated the `width` and `height` props from `ImageProps` I thought we should add them to the component so the user could still pass them.
On the other hand, you are right we can take the size and pass it as width and height.
Should we go on the size solution ?

Reply:
I think that if we can achieve the same result with less props it's better, especially if they are only applicable in specific situations.
It'll probably require a little more code, but I doubt it's a lot more code.